database_connection_setup.rb

```diff
@@ -0,0 +1,7 @@
+require './lib/database_connection'
+
+if ENV['ENVIRONMENT'] == 'test'
+  DatabaseConnection.setup('makersbnb_test')
+else
+  DatabaseConnection.setup('makersbnb')
```

Comment:
We should probably remove one of these files, as connect_to_database and database_connection_setup are the same file, except one is ENVIRONMENT and one is RACK_ENV